pipeline/inferrer/common/requirements.in

```diff
@@ -0,0 +1,9 @@
+fastapi==0.95.0
+gunicorn==20.1.0
+uvicorn==0.11.7
```

Comment:
These are the requirements that were present in all of the inferrers own requirements.in files